src/include/miopen/mlo_internal.hpp

```diff
@@ -673,7 +673,7 @@ struct mlo_construct_pooling2D : mlo_construct_activ_lrn_pooling_common
     int _pooling_method;
     miopenIndexType_t _index_type;
     miopenPoolingWorkspaceIndexMode_t _wsp_index;
-    int _NAN_option; // NOLINT (bugprone-reserved-identifier)
```

Comment:
@averinevg @atamazov  @shurale-nkn just curious, do we know why this NOLINT and others mentioned in https://github.com/ROCmSoftwarePlatform/MIOpen/issues/1322#issuecomment-995347664 are ignored by compiler and still generating warnings?

Reply:
cppcheck does not recognize NOLINT